website/docs/error-reference.md

```diff
@@ -446,13 +446,18 @@ intent to create a new type alias.)
 
 ## 5035
 
-> Method \_ is marked override but does not override anything
-
-No corresponding `overridable` was found.
-
-- Check the ancestors of the class defining the `override`.
-- Check that the file defining the `overridable` method has a sigil greater than
-  `ignore`.
+A method was marked `override`, but sorbet was unable to find a method in the
+class's ancestors that would be overridden. Ensure that the method being
+overridden exists in the ancestors of the class defining the `override` method,
+or remove `override` from the signature that's raising the error. See
+[Override Checking](override-checking) for more information about `override`.
+
+If the method exists in a
+[`# typed: ignore`](static#file-level-granularity-strictness-levels) file sorbet
```

Comment:
```suggestion
If the parent method definitely exists at runtime, it might be hidden in a
[`# typed: ignore`](static#file-level-granularity-strictness-levels) file. Sorbet
```